resources/dicts/events/ceremonies/ceremony_text.json

```diff
@@ -800,38 +800,91 @@
             "bold": [
                 "(prefix)paw may have butted heads against nearly every Clan member they have, but being able to argue was a blessing in a medicine cat, as m_c will have to continue shoving odd mixtures down everyone's throat. StarClan will be watching them with pride."
             ],
-            "calm": [],
-            "careful": [],
-            "charismatic": [],
-            "childish": [],
-            "cold": [],
-            "compassionate": [],
-            "confident": [],
-            "daring": [],
-            "empathetic": [],
+            "calm": [
+                "(prefix)paw has proven themsleves with one of the most invaluble traits a medicine cat can have, a level head under pressure. The stars of Silverpelt twinkle in joy and acceptance as m_c recives their full name and duites as a full medicine cat."
+            ],
+            "careful": [
+                "It is (prefix)paw's attention to detail that earns them their medicine cat name of m_c. They will serve the Clan long and well, with no one doubting their skill and knowledge of herbs they've spent so many moons learning."
+            ],
+            "charismatic": [
+                "(prefix)paw's charming attitude does as much for their patients as their skill does. They will continue to heal with words and herbs as m_c now that they are a full medicine cat of c_n."
+            ],
+            "childish": [
+                "Though their actions may come off as lighthearted, (prefix)paw has worked hard to earn their name of m_c, and mews that they'll protect c_n as a medicine cat. Though they start giggling right after they say it, causing a few purrs of amusement from onlookers both here, and here after."
```

Comment:
the last bit of this reads a little weird to me. "both here, and here after." I'm not sure what this means

Reply:
"here after" refers to the afterlife, I was making a pun but I can change it